structures/README.adoc

```diff
@@ -0,0 +1,54 @@
+= Automation structures
+include::../_style/render.adoc[]
+
+Before we start to describe roles, playbooks, etc, we need to decide which one we use for what.
+This section is meant for topics which are across multiple structures and don't fit nicely within one.
+
+.[big]#**Define which structure to use for which purpose**#
+[%collapsible]
+====
+Explanations::
+define for what you want to use roles, playbooks, potentially workflows (in Ansible Tower/AWX), and how to split the code you write.
+
+Rationale::
+especially when writing automation in a team, it is important to have a certain level of consistence and make sure everybody has the same understanding.
+By lack of doing so, your automation becomes unreadable and difficult to grasp for new members or even for existing members.
++
+This structure will also help you to have a consistent level of modelization so that re-usability becomes easier.
+Metaphorically speaking, only if stones have been cut at roughly the same size, can they be properly used to build a house.
+
+Examples::
+The following is only one example of how to structure your content but has proven robust enough on multiple occasions.
++
+.Structure of Automation
+image::ansible_structures.svg[a hierarchy of landscape type function and component]
```

Comment:
```suggestion
image::ansible_structures.svg[a hierarchy of landscape, type, function, and component]
```

Reply:
I had this but Asciidoctor interprets the commas for height and width parameters, the pictures disappears in PDFs (the HTML code is broken but you don't see it).